Makefile

```diff
@@ -282,39 +297,6 @@ $(MYGOBIN)/kubeval:
 		rm -rf $$d; \
 	)
 
-# linux only.
-# This is for testing an example plugin that uses helm to inflate a chart
-# for subsequent kustomization.
-# Don't want to add a hard dependence in go.mod file to helm.
-# Instead, download the binaries.
-$(MYGOBIN)/helmV2:
```

Comment:
IMO, we need to retain support for v2 for another year.
It's only a slightly different command.
We currently support both (albeit with bash).